src/node/rpc/notifierinterface.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <stdint.h>
+#include <vector>
+
+namespace ccf
```

Comment:
I split this since our app do not need access to the node interface but do need access to the notifier.